composer.json

```diff
@@ -28,8 +28,8 @@
         "liip/url-auto-converter-bundle": "~1.0",
         "nelmio/api-doc-bundle": "~2.3",
         "jms/serializer-bundle": "~0.13",
-        "friendsofsymfony/rest-bundle": "~1.0",
-        "friendsofsymfony/user-bundle": "~1.3"
+        "friendsofsymfony/rest-bundle": "~1.7@dev",
```

Comment:
I might release 1.7 quickly and move all the planned feature additions for FOSRestBundle 1.7 to 1.8